met-cron/sample.env

```diff
@@ -1,20 +1,96 @@
+# GDX MET CRON Configuration
+# For more information on these values, please see the documentation
+# or met-cron/config.py
 
-DATABASE_USERNAME=analytics
-DATABASE_PASSWORD=analytics
-DATABASE_NAME=met
-DATABASE_HOST=localhost
-DATABASE_PORT=5432
+# Changes Flask's run mode and the set of env vars are used to configure the app. You should not need to change this here.
+FLASK_ENV=development
 
-MET_DATABASE_USERNAME=met
-MET_DATABASE_PASSWORD=met
-MET_DB_NAME=met
-MET_DATABASE_HOST=localhost
-MET_DATABASE_PORT=5432
+USE_DEBUG=True # Enable a dev-friendly debug mode
+TESTING= # Handle errors normally (False) or raise exceptions (True)
 
+# Miscellaneous Settings
+SECRET_KEY="" # For Flask sessions. If unset, this value is randomized
+# disables certain checks for user permissions and tenant access. Buggy.
+IS_SINGLE_TENANT_ENVIRONMENT=false
+USE_TEST_KEYCLOAK_DOCKER=false
+USE_DOCKER_MOCK=false
+ENGAGEMENT_END_TIME="5 PM"
 
-MET_ADMIN_CLIENT_ID=met-admin
-MET_ADMIN_CLIENT_SECRET=<GENERATED_CLIENT_SECRET>
-JWT_OIDC_ISSUER=https://localhost:8080/auth/realms/met
+# Keycloak configuration.
+# Populate from 'GDX Modern Engagement Tools-installation-*.json'
+# https://bcgov.github.io/sso-requests
+KEYCLOAK_BASE_URL="" # auth-server-url
+KEYCLOAK_REALMNAME="" # realm
+MET_ADMIN_CLIENT_ID="" # resource
+MET_ADMIN_CLIENT_SECRET="" # credentials.secret
+KEYCLOAK_CONNECT_TIMEOUT="60"
+
+# JWT OIDC configuration for authentication
+# Populate from 'GDX MET web (public)-installation-*.json'
+JWT_OIDC_AUDIENCE="" # resource
+JWT_OIDC_ISSUER="" # default: constructed from base url and realm name
+JWT_OIDC_WELL_KNOWN_CONFIG="" # default: constructed from issuer
+JWT_OIDC_JWKS_URI=""          # default: constructed from issuer
+# Object path to access roles from JWT token
+# JWT_OIDC_ROLE_CLAIM=realm_access.roles # SSO schema
```

Comment:
We should remove this commented-out line if it's not being used

Reply:
Removed it, thanks.